plonk/src/proof_system/prover.rs

```diff
@@ -919,6 +921,29 @@ impl<E: PairingEngine> Prover<E> {
                 )
             })
             .collect();
+
+        // mask splitting polynomials
+        // t'(X) = t(X) - b_last + b_now * X^(n+2)
+        // with t'_lowest(X) = t_lowest(X) - 0 + b_now * X^(n+2)
+        // and t'_highest(X) = t_highest(X) - b_last
```

Comment:
Make notation consistent with the paper. `t'_lo`, `t'_mid`, `t'_high` are the polynomials before masking, `t_lo`, `t_mid`, `t_high` are the polynomials after masking.